panel/browser-panel.cpp

```diff
@@ -186,6 +188,34 @@ QCefWidgetInternal::~QCefWidgetInternal()
 	closeBrowser();
 }
 
+bool QCefWidgetInternal::eventFilter(QObject *object, QEvent *event)
+{
+#ifdef __APPLE__
+	UNUSED_PARAMETER(object);
+	UNUSED_PARAMETER(event);
+
+	return true;
+#else
+	// Return early event does not target the window wrapper or no browser instance is present
+	if (object != window_ || !cefBrowser) {
+		return true;
+	}
+
+	// Also return early if the event is not a "FocusIn" event, only necessary to check if the wrapper is targeted
+	if (event->type() != QEvent::FocusIn) {
+		return true;
+	}
+
+	CefRefPtr<CefBrowserHost> host{cefBrowser->GetHost()};
+
+	if (host) {
+		host->SetFocus(hasFocus);
```

Comment:
```suggestion
		host->SetFocus(hasFocus());
```

Most checks in Qt are functions.